packages/rest/src/lib/RequestManager.ts

```diff
@@ -406,7 +417,7 @@ export class RequestManager extends EventEmitter {
 
 		const fetchOptions = {
 			agent: this.agent,
-			body: finalBody,
+			body: finalBody ?? null,
```

Comment:
Won't this cause issues?

Reply:
Undici's `RequestInit` type doesn't include `undefined` as a value;

```
Type '{ agent: httpAgent; body: string | ArrayBuffer | AsyncIterable<Uint8Array> | Blob | FormData | Iterable<Uint8Array> | DataView | URLSearchParams | null | undefined; headers: Record<...>; method: RequestMethod; }' is not assignable to type 'RequestInit' with 'exactOptionalPropertyTypes: true'. Consider adding 'undefined' to the types of the target's properties.
  Types of property 'body' are incompatible.
    Type 'string | ArrayBuffer | AsyncIterable<Uint8Array> | Blob | FormData | Iterable<Uint8Array> | DataView | URLSearchParams | null | undefined' is not assignable to type 'BodyInit'.
      Type 'undefined' is not assignable to type 'BodyInit'.ts(2375)
[RequestManager.ts(324, 67): ]()The expected type comes from property 'fetchOptions' which is declared here on type '{ url: string; fetchOptions: RequestInit; }'
```

Reply:
It does but doesn't 🥲
Try setting it to the manual undefined property?

Reply:
fixed in 405b6f050fa0275572b9de8988807966cb1548dd